custom_typings/estree.d.ts

```diff
@@ -0,0 +1,411 @@
+declare module 'estree' {
```

Comment:
Is this not a good typescript definition for estree: https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/estree